client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -152,21 +152,14 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 					ullamco laboris nisi ut aliquip ex ea commodo consequat.
 				</div>
 			</animated.div>
-
-			<Portal>
-				<DialogBackdrop
-					visible={ showPageSelector }
-					className="design-selector__page-layout-backdrop"
-				/>
-			</Portal>
-
 			<Dialog
 				{ ...dialog }
+				modal={ false }
 				hide={ () => {
 					history.push( Step.DesignSelection );
 				} }
 				aria-labelledby="page-layout-selector__title"
-				hideOnClickOutside
+				hideOnClickOutside={ false }
```

Comment:
Do we need this to fix tabbing? It seems kinda convenient to be able to leave the page selector by clicking outside of it?

Reply:
>It seems kinda convenient to be able to leave the page selector by clicking outside of it?

I'm not sure about that. Imagine clicking at the white page area outside the design selector grid and be thrown at intent gather step. Since it's not visually resembling modal as much, I think we're better with explicit back buttons, either contextual in the content area or in the header.

Reply:
We need to be able to click outside on buttons in the header and whatnot. This is probably necessary.

One note, `hideOnClickOutside` sets the prop to boolean `true`. Assuming that's not the default, we can remove the prop rather than explicitly setting it to `false`:

```suggestion
```

Reply:
Aight, sounds good! 👍

Reply:
> Do we need this to fix tabbing? It seems kinda convenient to be able to leave the page selector by clicking outside of it?

Yes indeed, because now that back button, domain picker button and create new site button is clickable/tabbable, the `hideOnClickOutside` behaviour is removed.

> One note, hideOnClickOutside sets the prop to boolean true. Assuming that's not the default, we can remove the prop rather than explicitly setting it to false.

Yes, that's right. `hideOnClickOutside` has a default value of `true`, that's why this has to be set explicitly to `false`.